docs/_data/nav.yml

```diff
@@ -2,7 +2,6 @@ primary:
   - href: /usage/
   - href: /brand/
   - href: /content/
-  - href: /accessibility/
```

Comment:
do we want to leave in some other way for this page to be navigable from the design site?

Reply:
> do we want to leave in some other way for this page to be navigable from the design site?

I think the intention is to remove it completely from the design system website, and that the redirect exists largely to prevent dead links in case it's referenced elsewhere. @juliaelman can confirm though if this aligns to her expectations.

Reply:
@zachmargolis great question. Talked with Nick. We will be updating the design.login.gov footer to match the footer on login.gov so that the statement is added there and we have a consistency in branding.